.circleci/config.yml

```diff
@@ -0,0 +1 @@
+version: 2
```

Comment:
@jhermsmeier please see the conversation here: https://www.flowdock.com/app/rulemotion/r-operations/threads/5GW1zzD0MXZRYo22OPdW7PlOtes

CircleCI is the way to go now.
You don't need to set up anything specific in CircleCi, just log in with your github account.